prime-router/src/main/kotlin/Sender.kt

```diff
@@ -199,12 +192,15 @@ abstract class Sender(
  *  as the base Sender abstract class, although may be extended / modified in the future.
  */
 class FullELRSender : Sender {
+    var schemaName: String
```

Comment:
Why not add this to the base `Sender` class? It seems like both FullELR and Topic senders use it?

Reply:
Yep, makes sense, and we can just make it nullable or use `""` later if a different child class doesn't use it (though I don't expect additional child classes anyway), will be in an upcoming commit